eng/targets/Helix.props

```diff
@@ -5,7 +5,7 @@
   <ItemDefinitionGroup>
     <HelixContent>
       <CopyToBuildDirectory>Never</CopyToBuildDirectory>
-      <CopyToPublishDirectory>Always</CopyToPublishDirectory>
+      <CopyToPublishDirectory>PreserveNewest</CopyToPublishDirectory>
```

Comment:
Would have been nice to have a "copy if not already present", but that isn't an option 😞

Reply:
Does this mean the helix content version will override the project ones?  When I was looking at this yesterday you could try putting the default version as default.runner.json and just rename it in the runtests scripts if the directory doesn't have an xunit.runner.json

Reply:
Yeah, it'll override if it's newer. We could do the file rename if we'd like. However, I'm not sure if there is really any significant difference between the xunit.runner.json files per project.

Reply:
There are surprisingly, since there are a bunch of them, some of them set some of the app domain settings, so its probably worth doing the rename logic 

like https://github.com/dotnet/aspnetcore/blob/3e056db750d562826473f476e4f9b02ad76ffdd3/src/Servers/Kestrel/xunit.runner.json

Reply:
Aww man. Alright, will look into making that change